cmd/openshift-tuned.go

```diff
@@ -128,33 +146,53 @@ func tunedReload() {
 	}
 }
 
-func nodeLabelsGet(clientset *kubernetes.Clientset, nodeName string) (nodeLabels map[string]string) {
+func nodeLabelsGet(clientset *kubernetes.Clientset, nodeName string) map[string][]string {
 	node, err := clientset.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
-	if err != nil {
-		panic(err.Error())
-	}
 	if errors.IsNotFound(err) {
 		log.Printf("%s: node %s not found\n", PNAME, nodeName)
 	} else if statusError, isStatus := err.(*errors.StatusError); isStatus {
 		log.Printf("%s: error getting node %v\n", PNAME, statusError.ErrStatus.Message)
-	} else if err != nil {
+	}
+	if err != nil {
 		panic(err.Error())
 	}
 
-	return node.Labels
+	// This is not strictly necessary, but be consistent with podLabelsGet and return the same type
+	nodeLabels := map[string][]string{}
+	for key, value := range node.Labels {
+		nodeLabels[key] = append(nodeLabels[key], value)
```

Comment:
Why do the node labels just need `append()` vs the `insertUnique()` for the pod labels below?

Reply:
Good call